src/Store/Actions/Actions.js

```diff
@@ -43,6 +43,16 @@ export const fetchCveDetails = synopsis => ({
     }).then(result => result)
 });
 
+export const fetchCveClusterDetail = (cveId, setCveClusterData) => ({
+    type: ActionTypes.FETCH_CVE_DETAIL_TABLE,
+    meta: {
+        timestamp: new Date()
+    },
+    payload: new Promise(resolve => {
+        resolve(APIHelper.fetchExposedClusters(cveId));
+    }).then(result => setCveClusterData(result))
```

Comment:
Can we avoid involving state setters here? I'd be consistent with other actions which save the results in reducers (e.g., see src/Store/Reducers/CVEDetailsPageStore.js). Once it's in the reducer (store), you can read it from your component.